### PR TITLE
18 add missing tokens

### PR DESCRIPTION
**Pull request by 0xCardiE**

### CODE REVIEW

File: box/config.yml [hunk 1]
```diff
@@ -97,6 +96,7 @@ php_xdebug_idekey: PHPSTORM
 php_xdebug_max_nesting_level: 256
 php_xdebug_remote_port: "9000"
 php_memory_limit: "512M"
+php_sendmail_path: "/usr/sbin/sendmail -t -i"
```
Comment (dmiric) [inline box/config.yml]: We are not using sendmail. Why do we need this?

File: docroot/sites/default/default.settings.php [hunk 2]
```diff
@@ -86,7 +86,7 @@
  * );
```
Comment (dmiric) [inline docroot/sites/default/default.settings.php]: Why is this file here?

File: docroot/profiles/dv/modules/custom/activity/activity_logger/activity_logger.tokens.inc [hunk 3]
```diff
@@ -148,16 +158,54 @@ function activity_logger_tokens($type, $tokens, array $data, array $options, Bub
...
+          $active_profile = \Drupal::entityTypeManager()->getStorage('profile')->loadByUser($user, "organisation_profile");
+
+          // Designate the field we want to render.
+          $field_names = array('field_org_address', 'field_org_telephone', 'field_org_email');
+          $display = [
+            'label' => 'hidden',
+            'type' => 'field_address',
+          ];
+
+          $data = ""
```
Comment (dmiric) [inline docroot/profiles/dv/modules/custom/activity/activity_logger/activity_logger.tokens.inc]: This does not seem right. Line is missing semicolon.

File: docroot/profiles/dv/modules/custom/activity/activity_logger/activity_logger.tokens.inc [hunk 4]
```diff
@@ -55,6 +55,16 @@ function activity_logger_token_info() {
     'description' => t("ReplyTo Email"),
   ];
 
+  $activity['recipient-data'] = [
```
Comment (dmiric) [inline docroot/profiles/dv/modules/custom/activity/activity_logger/activity_logger.tokens.inc:58]: We need to find a better place for tokens specific for dv. For now I suggest we just moved dv specific tokens to dv_mail module.

**Review by dmiric — CHANGES_REQUESTED** [review]
Fix stuff in the comments, and check #61

File: docroot/profiles/dv/modules/custom/activity/activity_logger/activity_logger.tokens.inc (hunk 3 above)
Comment (0xCardiE) [reply docroot/profiles/dv/modules/custom/activity/activity_logger/activity_logger.tokens.inc]: typo, was in hurry :) thnx

File: docroot/profiles/dv/modules/custom/activity/activity_logger/activity_logger.tokens.inc (hunk 4 above)
Comment (0xCardiE) [reply docroot/profiles/dv/modules/custom/activity/activity_logger/activity_logger.tokens.inc:58]: Ok, reasoning is that this are tokens used for sending emails and code for email sending is in dv_mail module? Ok I can move it there.

File: box/config.yml (hunk 1 above)
Comment (0xCardiE) [reply box/config.yml]: I wanted to have real email sending and receiving it in gmail for testing.

By default VM uses mailhog and puts emails there. I wasnt aware of the problem with mailgun and having only one email listed to go through so I was looking for solution and thought maybe mailhog was the problem why my emails are not going out. Turns out that wasnt problem, this is not needed since mailgun is "overriding" mailhog anyway and it doesnt matter.

File: docroot/sites/default/default.settings.php (hunk 2 above)
Comment (0xCardiE) [reply docroot/sites/default/default.settings.php]: I removed it, think it got auto created with some composer updates.

File: docroot/profiles/dv/modules/custom/activity/activity_logger/activity_logger.tokens.inc (hunk 4 above)
Comment (0xCardiE) [reply docroot/profiles/dv/modules/custom/activity/activity_logger/activity_logger.tokens.inc:58]: done, tested, working.

**Review by dmiric — APPROVED** [review]
